compiler/luci/export/src/CircleTensorExporter.cpp

```diff
@@ -578,30 +578,48 @@ bool has_same_values(luci::CircleConst *lhs, luci::CircleConst *rhs)
   switch (lhs->dtype())
   {
     case loco::DataType::FLOAT32:
+      if (lhs->size<loco::DataType::FLOAT32>() != rhs->size<loco::DataType::FLOAT32>())
+        return false;
```

Comment:
This code looks very ugly. Isn't it possible to just check `lhs->compression() == rhs->compression()`?